### DISCUSSION TITLE
Is "Range Retrieval" (Active Directory) supported?

### DISCUSSION THREAD

**Discussion by TheConstructor** [dsbenghe/Novell.Directory.Ldap.NETStandard#253] [General]
I just stumbled across https://learn.microsoft.com/de-de/previous-versions/windows/desktop/ldap/searching-using-range-retrieval detailing how you can ask Active Directory to return more than 1500 values of an attribute e.g. `member`. Is this currently implemented/support?

**Comment by dsbenghe** [comment]
Not specifically via api as it is a AD only feature, but should be implementable on top of existing api

**Reply by TheConstructor** [reply]
Yeah, I think I found a way, but I wasn't sure, that I wasn't missing something
